Importer/Event/ConfigurationErrorEvent.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Netdudes\ImporterBundle\Importer\Event;
+
+use Netdudes\ImporterBundle\Importer\Error\ImporterErrorInterface;
+use Netdudes\ImporterBundle\Importer\Log\LogInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+class ConfigurationErrorEvent extends Event
```

Comment:
The implementation seem to be exactly the same as for `ApplicationErrorEvent`.
Do you think it makes sense to extract logic to some other, common error class?